torchvision/transforms/functional.py

```diff
@@ -577,3 +577,48 @@ def to_grayscale(img, num_output_channels=1):
         raise ValueError('num_output_channels should be either 1 or 3')
 
     return img
+
+
+def random_erasing(img, sl=0.02, sh=0.4, r1=0.3, value=0):
```

Comment:
we should rename this to be `erase` and pass in the randomly generated values `target_area` and `aspect_ratio` thus removing the randomness from this function.

The generation of these random parameters can be moved into `RandomErasing.get_params()` as is done for example in the [RandomCrop](https://github.com/pytorch/vision/blob/master/torchvision/transforms/transforms.py#L235) transform

Reply:
@alykhantejani  Thanks for your review, I have updated it following your advice.